py3status/modules/gcal.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+"""
+Display upcoming calendar events from Google Calendar.
+
+Configuration parameters:
+    agenda_days: show calendar events for number of days (default 1)
+    button_refresh: mouse button to refresh this module (default 2)
+    cache_timeout: refresh interval for this module (default 3600)
+    command: customize gcalcli to dictate what to do with the events
+        (default 'gcalcli agenda --nocolor')
+    format: display format for this module (default '{format_event}')
+    format_event: display format for events
+        (default '[\?if=is_date&color=#ff0 %a %b %d ]
+        [\?if=is_time&color=#0f0 %-I:%M %p ][\?color=#f00 {event}]')
+    format_separator: show separator only if more than one (default ', ')
+    remove: list of strings to remove from the output (default [])
```

Comment:
I still very strongly think that these formats are too complex/crazy for defaults, and are actively anti user.

`color=#0f0` this type of thing is just evil.  We should not be hard coding a colour in a format as it makes it a real pain to remove.  If people want colourful output they are welcome to it but forcing it on people feels wrong.

on the 7 AM issue to me this is crazy if the time shows as `7:15 AM` then I should also see `7:00 AM` this is about consistency which is important.

I still think 24hr time should be used in the default output as that is more technical and wider used/understood.